gmachine.go

```diff
@@ -93,6 +95,19 @@ func (g *GMachine) Run() {
 				}
 				fmt.Fprintf(g.Stderr, "%c", g.A)
 			}
+		case CMP:
+			value := g.Memory[g.P]
+			if value == g.A {
+				g.T = 1
+			}
+			g.P++
+		case JMP:
```

Comment:
`JMP` would indicate an absolute (unconditional jump), and we need that, so what about adding `JEQ` for a conditional jump if the `Z` flag is set?